pom.xml

```diff
@@ -164,13 +182,19 @@ all modules and does not describe the project structure per-subproject.
   <dependency>
     <groupId>org.junit.jupiter</groupId>
     <artifactId>junit-jupiter-api</artifactId>
-    <version>5.12.2</version>
+    <version>null</version>
     <scope>test</scope>
   </dependency>
   <dependency>
     <groupId>org.junit.jupiter</groupId>
     <artifactId>junit-jupiter-engine</artifactId>
-    <version>5.12.2</version>
+    <version>null</version>
+    <scope>test</scope>
+  </dependency>
+  <dependency>
+    <groupId>org.junit.jupiter</groupId>
+    <artifactId>junit-jupiter-params</artifactId>
+    <version>null</version>
```

Comment:
@alexander-yevsyukov please see the `null` versions for JUnit (it's here and above). These are suspicious.

Something is wrong with either the version detection or our configuration (which is even worse).

Reply:
This is because of the `platform(JUnit.bom)`. The license reporter probably does not understand it.

Reply:
Have a look at the `BomsPlugin` which was recently introduced. I'm trying to minimize the forces and make it all a bit more lazy and manageable.